cypress/e2e/OrangeHRM/approveLeaveRequest.ts

```diff
@@ -0,0 +1,37 @@
+import ApproveRequest from "../../support/helpers/approveRequestHelper";
+import loginPage from "../../support/pageObjects/loginPage";
+import LogoutPage from "../../support/pageObjects/logoutFunctionality";
+import SearchInTable from "../../support/pageObjects/assertionForScheduled";
+const loginObj: loginPage = new loginPage();
+
+let id: any;
+let leaveId: any;
+let userName: string;
+describe("OrangeHRM ", () => {
+  beforeEach(function () {
+    cy.visit("https://opensource-demo.orangehrmlive.com/");
+    loginObj.login("Admin", "admin123");
+  });
+
+  it("Scenario #1 : Approve Leave Request ", () => {
+    ApproveRequest.addNewEmployeeViaAPI().then((resolve) => {
```

Comment:
what do you mean by resolve? and you are in the same scope so no need to store it in a variable, same thing for the others below